### ISSUE TITLE
FPS drops and occasional crashing whist in the introduction/versus scene right before the games starts. (Vulkan related)

### ISSUE THREAD

**Issue by filipejunqueira** [ValveSoftware/Dota-2#1875]
#### Your system information
https://gist.github.com/filipejunqueira/7678cd2b1918304981150b80b227b74e

    Have you checked for system updates?: YES
    Are you using the latest stable video driver available for your system? YES
    Have you verified the game files?: YES

#### Please describe your issue in as much detail as possible:

As the title says there is a occasional crashing and reliable FPS drops when right before the game loads in the versus/introduction screen. It also reliably drops in the first few seconds of gameplay after the versus screen. 

This has happened in the past and got worse after the "watch tab update", likely to be related to vulkan since the issue happens only with vulkan enabled.

There is also a performance drop during gameplay (not necessary team fights) and in the in-game navigation menu.

#### Steps for reproducing this issue:

1. Find a game on Dota2.
2. Observe your FPS droping during the versus screen
3. FPS realibly drops in the first few seconds of gameplay after the versus screen. 
4. Open navigation menu

**Comment by stefanriest** [comment]
True, when you start the computer, start Dota 2, in game you have a nice FPS 120+, after some minutes it drops down to almost 1 fps, after 5 minutes it goes back up to like 15-20fps. SAME happens when you come out of standby mode. worst case you drop out of the game and the game is almost freezing.

my idea:
Maybe there is some endless Loop? or something similar
Maybe the CPU almost only is used?

It happens on Vulkan and OpenGL, the specs should be really fine.iMac 2017, 3,8ghz, 32g ram, radeon pro 580 8gb
